### PR TITLE
Feat/162/add UI kit radio checkbox

### PR DESCRIPTION
**Pull request by valfaro123**
### 🔥 Summary
- We want to add a radio button to replace those in Factoring Demo

### 😤 Problem / Goals
- We currently have a checkbox with the correct styles but want to add a radio button for the desired behavior 

### 🤓 Solution
- Modified the existing checkbox to behave as a radio button

### CODE REVIEW

File: src/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from '@emotion/styled';
 import React, { useState } from 'react';
 import { pxToRem } from '../../utils';
-import CheckboxSvg from '../../assets/svg/icons/checkbox.svg';
+import XSvg from '../../assets/svg/icons/x-icon.svg';
```
Comment (DorianKwan) [inline src/components/Checkbox/Checkbox.tsx:4]: Love it.

File: src/components/RadioButtons/RadioButtonBox/RadioButtonBox.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  defaultChecked,
+  disabled,
+  onChange,
+  value,
+  name,
+  required,
+}) => {
+  const handleClick = () => {
+    onChange?.();
+  };
```
Comment (DorianKwan) [inline src/components/RadioButtons/RadioButtonBox/RadioButtonBox.tsx]: This is an unnecessary wrapper at the moment.  We could just add this directly to the onClick

**Review by DorianKwan — APPROVED** [review]

File: src/components/RadioButtons/RadioButtonBox/RadioButtonBox.tsx (hunk 2 above)
Comment (valfaro123) [reply src/components/RadioButtons/RadioButtonBox/RadioButtonBox.tsx]: Good catch. Updated

Comment (DorianKwan) [reply src/components/RadioButtons/RadioButtonBox/RadioButtonBox.tsx]: Awesome work dude!

**Pull request merged**